ktor-io/common/src/io/ktor/utils/io/ByteWriteChannelOperations.kt

```diff
@@ -30,6 +30,26 @@ public suspend fun ByteWriteChannel.writeInt(value: Int) {
     flushIfNeeded()
 }
 
+/**
+ * Writes a 32-bit floating-point [value] to the current [ByteWriteChannel].
+ */
+@OptIn(InternalAPI::class)
+public suspend fun ByteWriteChannel.writeFloat(value: Float) {
+    writeBuffer.writeFloat(value)
+    flushIfNeeded()
+}
+
+/**
+ * Writes a 64-bit floating-point value to the current [ByteWriteChannel].
+ *
+ * @param value The floating-point value to be written to the channel.
```

Comment:
And one more place here
```suggestion
 * Writes a 64-bit floating-point [value] to the current [ByteWriteChannel].
```